ansible/_bootstrap.yaml

```diff
@@ -0,0 +1,15 @@
+---
+
+  - hosts: bastion node control-plane
+    # linear means do not use mitogen, which requires python
+    strategy: linear
+    any_errors_fatal: true
+    name: "Bootstrap Nodes"
+    gather_facts: no
+    become: yes
+    vars:
+      gather_reachability_data: false
+
+    roles:
+      - wait-ssh
+      - bootstrap-python
```

Comment:
Isn't this already handled somewhere in this repo? Other Ansible tasks are being already run.

Reply:
Sure it is, the idea was to get things working first, then assemble the pieces. Since I greatly overestimated the scope it should be easier to whittle things down now